canonical/lxd-ui
# Fix keyboard mappings in graphic console

## Done

- fix English and foreign language keyboard usage in graphic console. The event scancodes differ by browser, rely instead on the event code, which is stable for most common keys and also more complete.

Fixes #348

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - Test the graphic console on a desktop vm. Can be a new VM with ubuntu desktop > detail page > console. Open a terminal in it and start typing. The AltGr and all other keys should work as expected.
